index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prem Chand's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous"
+    referrerpolicy="no-referrer"
+  />
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+
+</head>
+<body>
+    <header class = "pb-2">
+        <nav class = "pt-3 pb-3">
+            <ul class = "unordered-list d-flex flex-row justify-content-between">
+                <li class = "my-logo-line mt-1"><a href="">My Logo</a></li>
+                <li><img src="Images/hamburger.png" alt="hamburger-menu" class = "hamburger-height-width"></li>
+            </ul>
+        </nav>
+    </header>
+    
+    <main class = "pb-2 pt-5">
+        <section class = "p-3">
+            <h1 class = "name-tag">I'm Prem Chand <br> Glad to see <br> you!</h1>
+            <p class = "bio-info">I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you 
+                like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <aside>
+                <h6><a href="">LET'S CONNECT</a></h6>
+                <ul class = "unordered-list d-flex flex-row p-0">
+                    <li><a href="https://twitter.com/Chandu1111112"><img src="Images/twitter.png" alt="twitter-icon" class = "icon-size"></a></li>
+                    <li><a href=""><img src="Images/linkdin-icon.png" alt="linkdin-icon" class = "icon-size"></a></li>
+                    <li><a href=""><img src="Images/Medium.png" alt="Medium-icon" class = "icon-size"></a></li>
+                    <li><a href="https://github.com/PremChand12"><img src="Images/github.png" alt="github-icon" class = "icon-size"></a></li>
+                    <li><a href=""><img src="Images/victory-symbol.png" alt="victory-symbol" class = "icon-size"></a></li>
+                </ul>
+            </aside>
+        </section>
```

Comment:
- [ ] [Optioanl] Well done on aligning the elements! 👏🌟 However, I noticed that in some places you're still using margin and padding for alignment. 😕 As per project requirements, I recommend utilizing Flexbox consistently throughout your code to align all elements. This will ensure a consistent and cohesive design across different screen sizes. Take some time to review your code and update it to use Flexbox for alignment purposes. This will enhance the structure and responsiveness of your project. Keep up the good work! 💪😊

![image](https://github.com/PremChand12/mobile-first-project/assets/55840999/f3b0aa10-2434-48a5-a2be-1cd8a8422e53)